crud/src/main/java/com/redhat/lightblue/crud/CRUDOperationContext.java

```diff
@@ -53,6 +53,8 @@
     private final CRUDOperation CRUDOperation;
     private final HookManager hookManager;
     private final ExecutionOptions executionOptions;
+    private final Set<String> documentVersions=new HashSet<>();
+    private boolean updateIfSame;
```

Comment:
I would call it updateIfCurrent or updateIfNotStale. Same is not as clear to me.